typescript/common-resolvers/src/utils/add-typename-to-data.ts

```diff
@@ -0,0 +1,21 @@
+export const addTypenameToData = <
+  TData extends Record<string, any>,
+  TTypename extends string
```

Comment:
Please don't use `any` in the Record values but either an `object` instead of the Record, a `Record<string, unknown>` or a less generic type (_code suggestion is partial_):

```suggestion
  TRecord extends object,
  TTypename extends string
```